paper-tabs.html

```diff
@@ -60,7 +60,7 @@
 
 @group Paper Elements
 @element paper-tabs
-@mixins Polymer.CoreResizable https://github.com/polymer/core-resizable
+@@applys Polymer.CoreResizable https://github.com/polymer/core-resizable
```

Comment:
Also, `core-resizable` -> `iron-resizable-behavior` ... or maybe just remove this line altogether because it lacks meaning in 0.8 world.